rocAL/source/pipeline/master_graph.cpp

```diff
@@ -93,31 +93,33 @@ MasterGraph::~MasterGraph() {
+                                                                                                                                                                                                                                                 _first_run(true),
+                                                                                                                                                                                                                                                 _processing(false),
+                                                                                                                                                                                                                                                 _prefetch_queue_depth(prefetch_queue_depth),
+                                                                                                                                                                                                                                                 _out_data_type(output_tensor_data_type),
 #if ENABLE_HIP
-                                                                                                                                                                                     _box_encoder_gpu(nullptr),
+                                                                                                                                                                                                                                                 _box_encoder_gpu(nullptr),
 #endif
-                                                                                                                                                                                     _rb_block_if_empty_time("Ring Buffer Block IF Empty Time"),
-                                                                                                                                                                                     _rb_block_if_full_time("Ring Buffer Block IF Full Time") {
+                                                                                                                                                                                                                                                 _rb_block_if_empty_time("Ring Buffer Block IF Empty Time"),
+                                                                                                                                                                                                                                                 _rb_block_if_full_time("Ring Buffer Block IF Full Time"),
+                                                                                                                                                                                                                                                _last_batch_policy(last_batch_policy),
```

Comment:
same as my comment before

Reply:
Removed the two internal variables and created a std::pair internal variable which is returned in last_batch_policy_info() function